docker/local/docker-compose.yml

```diff
@@ -23,7 +23,10 @@ services:
     - "9000:9000"
     - "9022:22"
     environment:
-    - DATABASE_HOST=db
+    - ALLOWED_HOSTS=*
+    - DATABASE_URL=mysql://root:admin@db:3306/phishtray
+    - REACT_APP_HOST_BACKEND=http://cybsafe.local:9000
+    - REACT_APP_HOST_FRONTEND=http://cybsafe.local:3000
```

Comment:
Should we add these env vars and their values in the prod docker-compose file while we're at it?

Reply:
Don't think this is the right time for the - prod will have its own method to set env vars. (Presumably a user interface) We shouldn't be storing env vars in a file for prod or anything that is not local.